funfact/backend/__init__.py

```diff
@@ -55,6 +59,8 @@ def use(backend: str):
     '''
     global _active_backend
     try:
+        if enable_x64 and backend == 'jax':
+            os.environ['JAX_ENABLE_X64'] = 'True'
```

Comment:
Can we also verify if x64 is enabled using the method as suggested by the JAX [docs](https://jax.readthedocs.io/en/latest/notebooks/Common_Gotchas_in_JAX.html):
> We can then confirm that x64 mode is enabled:
``` py
import jax.numpy as jnp
from jax import random
x = random.uniform(random.PRNGKey(0), (1000,), dtype=jnp.float64)
x.dtype # --> dtype('float64')
```

Reply:
I have added this to `_jax.py`. However, this warning will only trigger if JAX was already loaded separately from FunFact before with a different X64 setting. E.g.
```
import jax # load 32 bit
...
ff.use('jax', enable_x64=True) # warning will be shown here
```

 If the user does something like:
```
ff.use('jax', enable_x64=False) # load 32 bit
...
ff.use('jax', enable_x64=True) # try to load 64 bit
```
The second run does not actually get imported again presumably because in
```
_active_backend = getattr(
            importlib.import_module(f'funfact.backend._{backend}'), clsname
        )
```
`_jax.py` has already been loaded before. So the warning is not thrown in this case.

Reply:
I see. Sounds really tricky. How about we just stay as-is and assume that users will do things in the right order for the time being then? :cry:

Reply:
Ok, I have removed this again.